doc/examples.xml

```diff
@@ -309,3 +309,35 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="LollipopGraph">
+<ManSection>
+  <Oper Name="LollipopGraph" Arg="[filt, ]m, n"/>
+  <Returns>A digraph</Returns>
+  <Description>
+    If <A>m</A> and <A>n</A> are positive integers, 
+    then this operation returns the <A>(m, n)</A>-lollipop graph
+    that consists of a complete digraph on <A>m</A> vertices and a 
+    path digraph on  <A>n</A> vertices connected by a bridge.
+
+    From
+    <URL>https://en.wikipedia.org/wiki/Lollipop_graph</URL>
+    <P/>
```

Comment:
I think this is a better way of putting it: (in particular, in Digraphs we say "chain digraph" rather than "path digraph".
```suggestion
    then this operation returns the <E>(<A>m</A>, <A>n</A>)-lollipop graph</E>.
    As defined at <URL>https://en.wikipedia.org/wiki/Lollipop_graph</URL>,
    this consists of a complete digraph on <A>m</A> vertices and the
    symmetric closure of a chain digraph on <A>n</A> vertices,
    connected by a bridge. <P/>
```
It is also undefined what a 'bridge' is. I think it's okay to use the word bridge initially. But afterwards, in any case, I think we need another paragraph to explicitly say what the vertices and edges are, similar to what I suggested in https://github.com/digraphs/Digraphs/pull/423/files#r592305981

In particular, in my opinion, you should explicitly note that:
* the complete digraph (the 'body' of the lollipop) is on the vertices `[1 .. m]`.
* the 'stick' of the lollipop is on the vertices `[m+1 .. m+n]`.
* The two parts are joined by the edge `[m, m + 1]` and its reverse.